bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/Status.java

```diff
@@ -24,16 +24,16 @@
  */
 @NonNullByDefault
 public class Status {
-    public @NonNullByDefault({}) Double averageFuelConsumption;
+    public double averageFuelConsumption = -1;
```

Comment:
Maybe you can define and use constant (`UNDEFINED`) for the -1 value? It makes the code easier to read and it would also allow for easily updating all occurrences it in the future if one day you need to change it when there is a variable that can have negative values.

Reply:
Yes, you're completely right.